recipe/cudatoolkit-dev-post-install.py

```diff
@@ -277,7 +277,10 @@ def set_config():
     cudatoolkit["patch_url_ext"] = f""
 
     if sys.platform.startswith("win"):
-        cudatoolkit["blob"] = f'cuda_{cudatoolkit["version"]}_{cudatoolkit["driver_version"]}_win10.exe'
+        if(cudatoolkit["version"].startswith("11.6")):
```

Comment:
This should probably be a greater equal than check to avoid issues in future. You can use distutils.version.

Reply:
modification made